Description: Make makefile compatible with Debian's standards.
Author: Mateusz Łukasik <mati75@linuxmint.pl>
Last-Update: 2023-03-22

--- a/Makefile
+++ b/Makefile
@@ -1,5 +1,6 @@
-LIBS= `pkg-config --libs glib-2.0 gtk+-3.0 libmenu-cache`
-CFLAGS+= -g -Wall `pkg-config --cflags glib-2.0 gtk+-3.0 libmenu-cache`
+PKG_CONFIG ?= pkg-config
+LIBS= `$(PKG_CONFIG) --libs glib-2.0 gtk+-3.0 libmenu-cache`
+CFLAGS+= -g -Wall `$(PKG_CONFIG) --cflags glib-2.0 gtk+-3.0 libmenu-cache`
 CC=gcc
 #-DG_DISABLE_DEPRECATED
 
@@ -7,7 +8,7 @@ CC=gcc
 CFLAGS+=-DWITH_ICONS
 # Uncomment this line if Openbox can display SVG icons
 # Check SVG support with '$ ldd /usr/bin/openbox | grep svg', librsvg must appear..
-# CFLAGS+=-DWITH_SVG
+CFLAGS+=-DWITH_SVG
 
 prefix= /usr/local
 DESTDIR ?= $(prefix)
@@ -32,7 +33,6 @@ clean:
 	@rm -rf doc
 
 install:
-	@strip -s openbox-menu
 	@install -Dm 755 openbox-menu $(BINDIR)/openbox-menu
 
 doc:
