Description: Fix FTBFS with gcc-10
Author: Mateusz Łukasik <mati75@linuxmint.pl>
Bug-Debian: https://bugs.debian.org/957633
Last-Update: 2020-08-13

--- a/src/openbox-menu.h
+++ b/src/openbox-menu.h
@@ -83,7 +83,7 @@ gboolean context_get_persistent (OB_Menu
 void context_free(OB_Menu *);
 
 #ifdef WITH_ICONS
-	GtkIconTheme *icon_theme;
+	extern GtkIconTheme *icon_theme;
 #endif
 
 #endif // __OPENBOXMENU_APP__
--- a/src/utils.c
+++ b/src/utils.c
@@ -168,7 +168,7 @@ clean_exec (MenuCacheApp *app)
 
 #if WITH_ICONS
 
-extern GtkIconTheme *icon_theme;
+GtkIconTheme *icon_theme;
 
 /****f* utils/item_icon_path
  * OUTPUT
